### ISSUE TITLE
update translation files automatically

### ISSUE THREAD

**Issue by plata** [PhoenicisOrg/Scripts#480]
As we do not have a build system for the scripts repository, the translation files are usually not updated. We must find a way to update them automatically.

It should be possible to do this with Travis. However, there is no easy out-of-the-box solution, so it requires a script which is called from the `.travis.yml`.

Travis should be configured such that it runs whenever anything has been committed to master.

The required steps in the script are:
1. updated the translation files (see [wiki](https://github.com/PhoenicisOrg/Scripts/wiki/Translation))
2. if anything changed, create pull request (can be done with the GitHub API:  https://developer.github.com/v3/pulls/#create-a-pull-request)

**Comment by madoar** [comment]
This is an interesting idea.

Here some questions from me:
- to create a pull request I believe travis requires a github user account, right? Do you know how we can create such an account?
- I believe that we'll always need to create a new PR after a change, because the date of the translation files (i.e. the `.po` and `.pot` files) always changes, therefore I would suggest to limit the script execution to changes done to the master branch, i.e. only after a new PR has been merged
- in addition we would need to write some kind of script that does the following:
   1) create a new git branch on github
   2) apply the formatting on it
   3) push the changes to the branch
   4) create a new PR

**Comment by plata** [comment]
* I don't think we need an account. The access token should be enough. See e.g. [How to set up TravisCI for projects that push back to GitHub ](https://gist.github.com/willprice/e07efd73fb7f13f917ea)
* We must avoid this. I'm not yet really sure how. A possibility could be to get the number of changes, subtract the number of `.po` files + 1 (the `.pot`) and create a pull request only if this is >0. This should work because we now that we will have exactly 1 change in every `.po` and `.pot` if nothing changed (i.e. only the date changed).
* If Codacy works correctly, we should not need this because Codacy should detect bad formatting.
